e2e/components/Tooltip/Tooltip-test.avt.e2e.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('Tooltip @avt', () => {
+  test('accessibility-checker @avt - Tooltip default state', async ({
```

Comment:
LGTM! Maybe we call remove this `@avt` from the test, and keep only in the describe, right?
```suggestion
  test('accessibility-checker - Tooltip default state', async ({
```